docs/04-reference/wingsdk-spec.md

```diff
@@ -318,6 +359,15 @@ resource Bucket {
   inflight signed_url(key: str, duration?: duration): str;
 }
 
+struct FileMetadata {
+  /** The size of the file in bytes. */
+  size: Size;
```

Comment:
I was imagining `Size` as a standard type, similar to the one in the CDK ([link](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.Size.html)). I wonder if this should be a primitive type like `duration`? 🤷

Reply:
I definitely think we should have size literals, would be real nice

Reply:
Oh that would be cool